src/current/_includes/releases/v24.1/v24.1.0-alpha.2.md

```diff
@@ -0,0 +1,92 @@
+
+<h3 id="v24-1-0-alpha-2-sql-language-changes">SQL language changes</h3>
+
+- Added an option for node-level [sequence](https://www.cockroachlabs.com/docs/v23.2/serial) caching. All the sessions on the node can share the same cache, which can be concurrently accessed. The `serial_normalization` [session variable](https://www.cockroachlabs.com/docs/v23.2/set-vars#supported-variables) can now be set to the value `sql_sequence_cached_node`. If this value is set, the [cluster setting](https://www.cockroachlabs.com/docs/v23.2/cluster-settings) `sql.defaults.serial_sequences_cache_size` can be used to control the number of values to cache in a node, with a default of 256. The `PER NODE CACHE` sequence option (syntax is `[ [ PER NODE ] CACHE # ]`)  is now fully implemented and will allow nodes to cache sequence numbers. A cache size of 1 means there is no cache, and cache sizes of less than 1 are not valid. [#118546][#118546]
+
+<h3 id="v24-1-0-alpha-2-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that prevented the use of [PL/pgSQL](https://www.cockroachlabs.com/docs/v23.2/plpgsql) routines with complex variable names that require double quotes. This bug had existed since v23.2. [#119034][#119034]
+- Fixed a bug that could cause creation of a syntactically invalid [PL/pgSQL](https://www.cockroachlabs.com/docs/v23.2/plpgsql) routine to return the wrong error. This bug had existed since v23.2. [#119034][#119034]
+- Fixed a bug that could result in a syntax error if a [PL/pgSQL](https://www.cockroachlabs.com/docs/v23.2/plpgsql) routine was created with an escaped string constant in the routine body. This bug had existed since v23.2. [#119034][#119034]
+- Fix a bug where running [`RESTORE`](https://www.cockroachlabs.com/docs/v23.2/restore) on certain backups would open a very large number of connections to the backup storage provider. [#119840][#119840]
+- If a user has the `VIEWACTIVITYREDACTED` [privilege](https://www.cockroachlabs.com/docs/v23.2/security-reference/authorization#managing-privileges), then they should not be able to see constants inside of queries that originate from other [users](https://www.cockroachlabs.com/docs/v23.2/security-reference/authorization#roles) in the [`SHOW SESSIONS`](https://www.cockroachlabs.com/docs/v23.2/show-sessions) output. Previously, this redaction was not occurring, but this is now fixed. [#119820][#119820]
+- The [`SHOW QUERIES`](https://www.cockroachlabs.com/docs/v23.2/show-queries) and [`SHOW STATEMENTS`](https://www.cockroachlabs.com/docs/v23.2/show-statements) commands were incorrectly requiring that the user had the `VIEWACTIVITY` or `VIEWACTIVITYREDACTED` [privilege](https://www.cockroachlabs.com/docs/v23.2/security-reference/authorization#managing-privileges). However, a user should always be able to view their own queries, even without this privilege. This is now fixed. [#119820][#119820]
```

Comment:
```suggestion
- Previously, the [`SHOW QUERIES`](https://www.cockroachlabs.com/docs/v23.2/show-queries) and [`SHOW STATEMENTS`](https://www.cockroachlabs.com/docs/v23.2/show-statements) commands incorrectly required the user to have the `VIEWACTIVITY` or `VIEWACTIVITYREDACTED` [privilege](https://www.cockroachlabs.com/docs/v23.2/security-reference/authorization#managing-privileges). This is now fixed, as a user should always be able to view their own queries. [#119820][#119820]
```

Reply:
✅  (kept the phrase "even without this privilege" since I think it makes it even clearer)